app/controllers/dashboard_controller.rb

```diff
@@ -16,6 +16,7 @@ def show
       @draft_works << work if version_status.draft? && work.user == current_user
       [work.id, version_status]
     end
+    @your_collections = current_user.your_collections.to_a.sort_by!(&:title)
```

Comment:
Unfortunately the underlying query is too complicated to just slap an `order` on it.